CHANGELOG/CHANGELOG-1.14.md

```diff
@@ -0,0 +1,88 @@
+- Update `Install-WindowsCNI-Containerd.ps1` script to make it compatible with containerd 1.7. ([#5528](https://github.com/antrea-io/antrea/pull/5528), [@NamanAg30])
+- Add checksum of Windows config files as annotations to Windows manifests to avoid stale ConfigMaps. ([#5545](https://github.com/antrea-io/antrea/pull/5545), [@Atish-iaf])
+- Add a ClusterSet deletion webhook for the leader cluster to reject ClusterSet deletion if there is any MemberClusterAnnounce. ([#5475](https://github.com/antrea-io/antrea/pull/5475), [@luolanzone])
+- Add a new all-in-one manifest for the Multi-cluster leader cluster, and clarify the version of Multi-cluster user guide for the new leader manifest. ([#5389](https://github.com/antrea-io/antrea/pull/5389) [#5531](https://github.com/antrea-io/antrea/pull/5531), [@luolanzone])
+- Clean up auto-generated resources in leader and member clusters when a ClusterSet is removed. ([#5351](https://github.com/antrea-io/antrea/pull/5351), [@luolanzone])
+- Recreate resources when a member cluster rejoins the ClusterSet. ([#5410](https://github.com/antrea-io/antrea/pull/5410), [@luolanzone])
+
+### Changed
+
+- Multiple APIs are promoted from beta to GA. The corresponding feature gates are removed from antrea config files.
+  - Promote feature gate EndpointSlice to GA. ([#5393](https://github.com/antrea-io/antrea/pull/5393), [@hongliangl])
+  - Promote feature gate NodePortLocal to GA. ([#5491](https://github.com/antrea-io/antrea/pull/5491), [@hjiajing])
+  - Promote feature gate AntreaProxy to GA. ([#5401](https://github.com/antrea-io/antrea/pull/5401), [@hongliangl])
+- Make antrea-controller not tolerate Node unreachable to speed up the failover process. ([#5521](https://github.com/antrea-io/antrea/pull/5521), [@tnqn])
+- Improve `antctl get featuregates` output. ([#5314](https://github.com/antrea-io/antrea/pull/5314), [@cr7258])
+- Increase the rate limit setting of `PacketInMeter` and the size of `PacketInQueue`. ([#5460](https://github.com/antrea-io/antrea/pull/5460), [@GraysonWu])
+- Add `hostAliases` to Helm values for Flow Aggregator. ([#5386](https://github.com/antrea-io/antrea/pull/5386), [@yuntanghsu])
+- Decouple Audit logging from AntreaPolicy feature gate to enable logging for NetworkPolicy when AntreaPolicy is disabled. ([#5352](https://github.com/antrea-io/antrea/pull/5352), [@qiyueyao])
+- Switch Traceflow CRD validation to webhook validation. ([#5230](https://github.com/antrea-io/antrea/pull/5230), [@shi0rik0])
+- Stop using `/bin/sh` and invoke the binary directly for OVS commands in Antrea Agent. ([#5364](https://github.com/antrea-io/antrea/pull/5364), [@antoninbas])
+- Install flows for nested Services in `EndpointDNAT` only when Antrea Multi-cluster is enabled. ([#5411](https://github.com/antrea-io/antrea/pull/5411), [@hongliangl])
+- Make rate-limit of features configurable to help with performance tuning. ([#5450](https://github.com/antrea-io/antrea/pull/5450), [@GraysonWu])
```

Comment:
This a bit too vague. We should specify that this is for PacketIn.

> Make rate-limiting of PacketIn messages configurable; the same rate-limit value applies to each feature that is dependent on PacketIn messages (e.g, Traceflow) but the limit is enforced independently for each feature.

Reply:
Done